phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -85,10 +85,14 @@
 import java.util.concurrent.atomic.AtomicInteger;
 
 import static org.apache.phoenix.mapreduce.PhoenixJobCounters.INPUT_RECORDS;
+import static org.apache.phoenix.mapreduce.index.PhoenixIndexToolJobCounters.AFTER_INVALID_DETAIL_INFO_INDEX_HAS_EXTRA_CELLS_COUNT;
```

Comment:
Nit: Prefer to keep the same prefix as others:. e.g. AFTER_REBUILD_INVALID_INDEX_ROW_COUNT_COZ_EXTRA_CELLS, AFTER_REBUILD_INVALID_INDEX_ROW_COUNT_COZ_MISSING_CELLS
I used 'coz' to keep it shorter .. and also express reason or invalidity.